components/EventInformation.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Link from 'next/link';
+import dayjs from 'dayjs';
+import { prependHttp } from '../utils/helpers';
+
+
+const EventInformation = ({ event, user, start, dateFormat, end, duration, loadError, myTickets, isAuthenticated, attendees, attendEvent, featured, featureEvent }) => {
```

Comment:
attendEvent, featureEvent, loadError etc should be defined in here instead of being imported.
And let's rename this component to EventActions and just have the list of actions instead of all this.
Then the event dates could be put into another component called EventDates or something that can be re-used.